prombench/manifests/prombench/benchmark/7_prometheus_remote_write.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: prometheus-sink
+  namespace: prombench-{{ .PR_NUMBER }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sink
+  template:
+    metadata:
+      namespace: prombench-{{ .PR_NUMBER }}
+      labels:
+        app: sink
+    spec:
+      containers:
+      - name: prom-sink
+        image: quay.io/bwplotka/sink:latest
+        resources:
+          limits:
+            cpu: "1"
+            memory: "350Mi"
```

Comment:
Leave off limits until we know more about what it needs.  
If it hits the memory limit it will crash, which will be annoying.